### PR TITLE
Fixing issue with .travis.yml that was found from failed build

### PR DESCRIPTION
**Pull request by jmhossler**
https://travis-ci.org/Roastd/roastd/builds/360077581#L2840

This was the failing build. The PR was merged before a success was confirmed after making changes. This should not happen. This PR (and PRs after this is merged in) should pass the travis-ci job before being merged. Anything that isn't should be immediately reverted.

Also, I think we should move to 'squash and merge' instead of just merge. This makes it easier to revert changes in master when an issue like this occurs, since master would have just one bad commit instead of all the commits, and the commit summary is the PR name (which should explain what was done).

### CODE REVIEW

**Comment by tlmurphy** [comment]
:rip:

**Comment by jmhossler** [comment]
The main issue I'm having right now is getting the tests passing. It's not the same issue as before, and it seems it's mainly the combination of emulator/api_version that is causing issues.

**Comment by jmhossler** [comment]
ayyy ;)

**Comment by tlmurphy** [comment]
image

File: .travis.yml [hunk 1]
```diff
@@ -11,8 +10,8 @@ cache:
  - $HOME/.gradle/wrapper/
 env:
   global:
-   - ANDROID_API_LEVEL=27
-   - EMULATOR_API_LEVEL=27
+   - ANDROID_API_LEVEL=26
```
Comment (jmhossler) [inline .travis.yml]: I could not get 27 and 23 working together, but I could get 26 and 23 working.

**Pull request merged**